src/ts_catalog/compression_settings.c

```diff
@@ -151,22 +166,28 @@ ts_compression_settings_get(Oid relid)
 	return settings;
 }
 
-TSDLLEXPORT bool
-ts_compression_settings_delete(Oid relid)
+TSDLLEXPORT CompressionSettings *
+ts_compression_settings_get(Oid relid)
+{
+	return compression_settings_get(relid, false);
+}
+
+TSDLLEXPORT CompressionSettings *
+ts_compression_settings_get_by_compress_relid(Oid relid)
```

Comment:
Add a todo to make clear this will eventually go away since its an intermediate step towards the complete refactor.